app/lib/dttp/code_sets/degree_subjects.rb

```diff
@@ -0,0 +1,1102 @@
+# frozen_string_literal: true
+
+module Dttp
+  module CodeSets
+    module DegreeSubjects
+      MAPPING = {
```

Comment:
@gpeng this feels like it going to be error prone. Maybe we should do a http call to seed this when empty or on start up.

Reply:
There are duplicates in DTTP, so Ben has to decide which ones to use. It looks like we'll have to fill in this manually once Ben compiles the list. It's a shame.

Reply:
Discussed with with @defong. It is what it is... We have two sources of data at the moment and both are 'correct' for their domain. For now we need to map between the two which isn't ideal but it's a step on the way to where we want to go so 🤷